ReactWindows/ReactNative.Net46/ReactNative.Net46.csproj

```diff
@@ -61,8 +61,14 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="PresentationCore" />
+    <Reference Include="PresentationFramework" />
+    <Reference Include="SocketMessaging, Version=1.1.6002.26530, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\SocketMessaging.1.1.6002.26530\lib\net451\SocketMessaging.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
```

Comment:
Nitpick, but the .NET Framework dependencies (i.e., those that don't come from NuGet or private drops, or those without HintPaths) typically come above the rest of the dependencies here.